webapp/channels/src/components/admin_console/system_user_detail/system_user_detail.tsx

```diff
@@ -383,7 +383,7 @@ export default class SystemUserDetail extends React.PureComponent<Props & RouteC
                             body={
                                 <React.Fragment>
                                     <span className='SystemUserDetail__position'>{user.position}</span>
-                                    <span className='SystemUserDetail__field-label'>{Utils.localizeMessage('admin.userManagement.userDetail.email', 'Email')}</span>
+                                    <span className='SystemUserDetail__field-label'>{this.props.intl.formatMessage({id: 'admin.userManagement.userDetail.email', defaultMessage: 'Email'})}</span>
```

Comment:
Yeah, please do. I mostly want to avoid `injectIntl` when possible since it's clunky and can cause some other problems like with multiple exports and refs. I wouldn't care as much if this could use `useIntl`.

In general, I've been thinking of pushing more for `FormattedMessage` wherever possible since it seems more idiomatic for React, but I haven't really decided if I care enough to convince people of that, so I haven't bothered pushing for it 😛

Reply:
There are many places in this file that are requiring strings as children, so for now I will leave it like this. All this should be reviewed and moved to functional components either way...